website/content/docs/secrets/aws.mdx

```diff
@@ -474,6 +474,40 @@ have a time-to-live of 1 hour and automatically refresh when they expire.
 Please see the [API documentation](/vault/api-docs/secret/aws#configure-root-credentials)
 for more details on the fields associated with plugin WIF.
 
+## Cross-account static role management
+
+<EnterpriseAlert product="vault" />
+
+Vault supports cross-account access for AWS static roles. You can configure credential management for IAM users across multiple AWS accounts by assuming roles in the other accounts.
+
+To configure a cross-account static role:
+
+1. Ensure Vault has access to AWS STS and IAM endpoints.
+1. Create an IAM role in the target AWS account that allows Vault to assume it.
+  1. The role must trust the Vault AWS account to assume it and have the necessary IAM permissions to manage IAM users.
+  1. Use an external ID for enhanced security (optional but recommended).
+1. Create a static role in Vault.
+  1. Set the target account's ARN with the `assume_role_arn` field.
+  1. Set the target account's session name with the `assume_role_session_name` field.
+
+```shell-session
+$ vault write aws/static-roles/<role-name> \
+    username="<iam-user-to-manage>" \
+    assume_role_arn="arn:aws:iam::<account-id>:role/<role-name-in-target-account>" \
+    assume_role_session_name="<session-name>" \
+    external_id="<unique-id>" \
+    rotation_period="1h"
+```
+
+Once configured, Vault will:
```

Comment:
As written, we're implying these steps happen as soon as the configuration takes hold. Is that true?

Reply:
Yes, and returns IAM user's access key and secret key when requested via `aws/static-creds/<role-name>`.